src/node/src/node.rs

```diff
@@ -75,16 +75,40 @@ pub fn start() {
                                 state.store(STOPPED, Ordering::SeqCst);
                                 return;
                             }
+                            Ok(Ok(Command::Bind(
+                                portal_name,
+                                statement_name,
+                                param_formats,
+                                raw_params,
+                                result_formats,
+                            ))) => {
+                                match query_executor.bind_prepared_statement_to_portal(
```

Comment:
I think we need to introduce one more optional stage in query processing pipeline for prepared statement and move this function to separate struct.

No need to change anything for this comment.